test/propagation_test.cpp

```diff
@@ -227,15 +243,21 @@ TEST_CASE("Binary Span Context") {
       REQUIRE(received_context->id() == 420);
       REQUIRE(received_context->traceId() == 123);
       if (priority_sampling) {
+        // REQUIRE(received_context->origin() == origin);
```

Comment:
What's with the commented code?

Reply:
I started out modifying the existing tests then decided to change to doing a set of specific tests. Forgot to go remove those things.